packages/@azure/servicebus/data-plane/examples/typescript/advancedFeatures/deferral.ts

```diff
@@ -101,7 +101,7 @@ async function receiveMessage(): Promise<void> {
     let receiver = receiveClient.getReceiver();
     receiver.receive(onMessage, onError, { autoComplete: false }); // Disabling autoComplete so we can control when message can be completed, deferred or deadlettered
     await delay(10000);
-    receiver.close();
+    await receiver.close();
```

Comment:
Shouldnt this be done in the javascript sample as well?

Reply:
Yes, should be.